getGit.tex

```diff
@@ -154,23 +185,12 @@ \section{What, Why, and How?}
 \begin{frame}
     \frametitle{Git and Operating Systems}
     \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
-        \item{Mac: ???}
-    \end{itemize}
-    \vspace{1cm} % generate some space between title and content
-\end{frame}
-
-\begin{frame}
-    \frametitle{Git and Operating Systems}
-    \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
+        \item{Windows: Install gitbash.}
+        \item{Linux: Install through package manager.}
         \item{Mac: ???}
```

Comment:
@turanoo What's the install process like for git on a mac?

Reply:
Not sure, I'll ask one of my friends with a Mac if he's willing to install git on it. I can't imagine it's very hard, but I don't have access to a Mac to try